arm-advisor/2017-03-31/swagger/advisor.json

```diff
@@ -0,0 +1,679 @@
+						"required": true,
+						"type": "string",
+						"format": "uuid"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"202": {
+						"description": "Recommendation generation is in progress."
+					},
+					"204": {
+						"description": "Recommendation generation has been completed."
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"PollGenerateRecommendationsStatus": {
+						"$ref": "../examples/PollGenerateRecommendationsStatus.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/recommendations": {
+			"get": {
+				"tags": [
+					"GetRecommendations"
+				],
+				"description": "Obtains cached recommendations for a subscription. The recommendations are generated or computed by invoking generateRecommendations.",
+				"operationId": "Recommendations_List",
+				"parameters": [
+					{
+						"$ref": "#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					},
+					{
+						"name": "$filter",
+						"in": "query",
+						"description": "The filter to apply to the recommendations.",
+						"required": false,
+						"type": "string"
+					},
+					{
+						"name": "$top",
+						"in": "query",
+						"description": "The number of recommendations per page if a paged version of this API is being used.",
+						"required": false,
+						"type": "integer"
+					},
+					{
+						"name": "$skipToken",
+						"in": "query",
+						"description": "The page-continuation token to use with a paged version of this API.",
+						"required": false,
+						"type": "string"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK.",
+						"schema": {
+							"$ref": "#/definitions/ResourceRecommendationBaseListResult"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				},
+				"x-ms-examples": {
+					"ListRecommendations": {
+						"$ref": "../examples/ListRecommendations.json"
+					}
+				}
+			}
+		},
+		"/providers/Microsoft.Advisor/operations": {
+			"get": {
+				"tags": [
+					"Operations"
+				],
+				"description": "Lists all the available Advisor REST API operations.",
+				"operationId": "Operations_List",
+				"parameters": [
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK.",
+						"schema": {
+							"$ref": "#/definitions/OperationEntityListResult"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			}
+		},
+		"/{resourceUri}/providers/Microsoft.Advisor/recommendations/{recommendationId}/suppressions/{name}": {
+			"get": {
+				"tags": [
+					"Suppressions"
+				],
+				"description": "Obtains the details of a suppression.",
+				"operationId": "Suppressions_Get",
```

Comment:
Does this operation provide a list of suppressions for one recommendations? In that case, the recommendation is to use the id Suppressions_ListByRecommendations